ivy_tests/test_ivy/test_frontends/test_tensorflow/test_linalg.py

```diff
@@ -246,3 +246,33 @@ def test_tensorflow_pinv(
         validate_args=False,
         name=None
     )
+
+
+)
```

Comment:
You have to add the `@given` decorator here and have to figure out a way to generate the required values. The best way is to implement a `composite function` or to use an already implemented one. I've also attached the link to the docs, kindly refer to the docs for more detail.
https://lets-unify.ai/ivy/deep_dive/15_ivy_frontends.html
https://lets-unify.ai/ivy/deep_dive/16_ivy_frontends_tests.html